src/integrationTest/java/uk/gov/hmcts/reform/laubackend/idam/bdd/UserDeletionAuditGetSteps.java

```diff
@@ -74,5 +74,15 @@ public void responseIsReturned(int statusCode, String status) {
 
     }
 
+    @Then("multiple deleted account entries are returned with size {int}")
+    public void multipleAccountEntriesReturnedWithCount(Integer count)
+        throws InvocationTargetException, IllegalAccessException, NoSuchMethodException {
+        final UserDeletionGetResponse response = jsonReader
+            .fromJson(responseBodyString, UserDeletionGetResponse.class);
+        int size = response.getDeletionLogs().size();
+        assertEquals(count, size, "Size does not match");
```

Comment:
You need to verify the order of the records too

Reply:
"Size does not match" is the opposite of the assertions

Reply:
You are not testing the negative scenarious

Reply:
New Tess have been added to check for direction and negative scenarios